docs/onboarding/configuration.md

```diff
@@ -58,6 +58,7 @@ You can change the working directory by passing the flag `--datadir` when runnin
 | nodeWorkers          | The number of threads to use for workers. A value of -1 will use the maximum possible amount of threads.
 | p2pSimulateLatency   | Randomly delay outbound messages up to this value |
 | peerPort             | Port on which to host the WebSocket listen server |
+| poolName             | Name to use for mining pool (optional) |
```

Comment:
Not a blocker, but in the future could this page be automated from `ironfish --help`'s output?

Reply:
Probably. I'll look into that for a future PR.